x-pack/plugins/event_log/server/es/names.ts

```diff
@@ -17,7 +17,11 @@ export interface EsNames {
   indexTemplate: string;
 }
 
-export function getEsNames(baseName: string, kibanaVersion: string): EsNames {
+export function getEsNames(
+  baseName: string,
+  kibanaVersion: string,
+  suffix: string = EVENT_LOG_NAME_SUFFIX
```

Comment:
Looks like @pmuellr mentioned this in his comment too, but I think the verdict is still out on if we'll need to include the kibana `space_name` in the index name as well. We currently do this on the `security-solution` side (e.g `.siem-signals-default` for the default space) so users can restrict access via ES Index Privileges. Nothing says we can't just include the space name in the `baseName` for the security-solution registry, but something I wanted to note. Will add an item to the alerts as data agenda for us to discuss _one last time_ 😉.

Reply:
no objections from my side certainly!

Reply:
I think the intention is to set `eventLogName` to `${baseName}${suffix}` instead of the existing `${baseName}${EVENT_LOG_NAME_SUFFIX}`.  Or I'm not understanding how the suffix is used.

And I guess the intention is to be able to create new versions of the event log structure, with different names.  We'll need to make sure we got all the index pattern, alias, ILM policy right with this, so should add some additional tests when suffix is passed in.  Jest tests (should be easy, I think), and I think some FT, where we'd have a server-side fixture create a new "suffixed" event log, write to it, make sure we can read from the alias, etc.

And given the scope of this, perhaps `suffix` is a bad name - it's actually kind of the "base name".

Also not clear what we'll be using this for - is this going to be what we use for the rule registry data writing target index?  For some reason I thought we were going with a similar, but different kind of "event log" story.

Reply:
I think these changes are now longer necessary - I now initialize the es names in the rule registry itself.